src/browser/modules/Stream/CypherScriptFrame/CypherScriptFrame.tsx

```diff
@@ -102,9 +102,8 @@ function CypherScriptFrame({
       </ContentSizer>
     </WrapperCenter>
   )
-  return (
-    <FrameTemplate className="no-padding" header={frame} contents={contents} />
-  )
+  // This wanted to disable padding on the template. will have to do it in other way than classname.
```

Comment:
Looks like you've already addressed this in the StyledComponents logic, so this comment can be removed?

Reply:
yes, good catch